gradle/libs.versions.toml

```diff
@@ -50,17 +59,24 @@ jackson-afterburner = { module = 'com.fasterxml.jackson.module:jackson-module-af
 jetty-server = { module = 'org.eclipse.jetty:jetty-server', version.ref = 'jetty' }
 jcl-over-slf4j = { module = 'org.slf4j:jcl-over-slf4j', version.ref = 'slf4j' }
 junit-jupiter-engine = { module = 'org.junit.jupiter:junit-jupiter-engine' }
+junit-jupiter-api = { module = 'org.junit.jupiter:junit-jupiter-api' }
 junit-platform-engine = { module = "org.junit.platform:junit-platform-suite-engine" }
+aws-java-sdk-lambda = { module = 'com.amazonaws:aws-java-sdk-lambda' }
```

Comment:
thought: I wonder if we should make this one a managed dependency too, even if it's an `implementation` dependency...

Reply:
I don't think users add them to their application but I am ok making it a managed dependency.

Reply:
Right, let's not make it managed.